pages/learn/manage/account.md

```diff
@@ -62,14 +62,14 @@ When an application needs to be shared with more than one user, the application
 
 | Member Type | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
 | ------------- |:-------------:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:|
-| Owner      | Yes      | Yes  | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
+| Administrator | Yes      | Yes  | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
 | Observer   | No       | No   | No   | No   | No   | No  | No  | No  | No  | No  |
 | Operator   | No       | No   | No   | Yes  | Yes  | Yes | Yes | No  | No  | No  |
 | Developer  | No       | No   | No   | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
 
-#### Owner
+#### Administrator
 
-The application owner is the user who first creates an application. The owner is the only user who can add other application members or delete the application.
+The Organization's [Administrators][orgAdministrator] are the users who first create an application. They are the only users who can add other application members or delete the application.
```

Comment:
```suggestion
In an application's scope, an [administrator][administrator] is a user who created the application. There can be multiple administrators in an application. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator's role](adminstrator) in an organization's scope. 
```

Reply:
Well there can obviously be administrators who have never created an app.
How about the one I just pushed?

Reply:
Technically we could just drop this from here, and also drop some of the columns of from the table, but part of me likes how it compares what app members can do, vs what administrators only can do.

Reply:
Yeah true that on the first point. 
That part of you that likes "that this is on the table" is right. From an application scope, I think we should explain the role of admins and that's what this does.